git-cliff-core/Cargo.toml

```diff
@@ -77,6 +77,7 @@ url.workspace = true
 dyn-clone = "1.0.17"
 urlencoding = "2.1.3"
 cacache = { version = "13.0.0", features = ["mmap"], default-features = false }
+chrono = "0.4.38"
```

Comment:
Can we use the [`time`](https://docs.rs/time/) crate directly instead of chrono? If I understood correctly, it is only used for converting a Rfc3339 string to a timestamp. `time` also [supports that](https://time-rs.github.io/api/time/format_description/well_known/struct.Rfc3339.html) and it should be more lightweight.

See: https://stackoverflow.com/questions/74935683/convert-utc-rfc3339-timestamp-to-local-time-with-the-time-crate

Thoughts?